deploy/cr/expose/cr_route.yaml

```diff
@@ -6,3 +6,4 @@ spec:
   replicas: 2
   expose:
     type: Route
+    host: http://example.org/
```

Comment:
@oraNod host must conform to DNS 952 subdomain conventions, i.e. www.example.org

Reply:
@dmvolod great catch. thanks. I've updated.